### ISSUE TITLE
Internal Server Error when used with Yoast 20.8 (released 23/05/23)

### ISSUE THREAD

**Issue by RichElse** [ashhitch/wp-graphql-yoast-seo#159]
Hi! Yoast released a new version yesterday since then WP GraphQL Yoast SEO fails with `Internal Server Error`.

```ApolloError: Internal server error
 {
graphQLErrors: [
{
message: 'Internal server error',
extensions: [Object],
locations: [Array],
path: [Array]
}
],
protocolErrors: [],
clientErrors: [],
networkError: null,
extraInfo: undefined
}
```
Yoast is a managed plug-on on WordPress hosted sites so not possible to rollback to a previous version of the Yoast plug-in. Diagnosed in WP by using GQL Explorer and removing `seo` fields until query successful. Temporary fix in headless implementation was to remove SEO object GQL query.

**Comment by mcourville** [comment]
+1, having the same issue:

`[GQtyError: Internal server error] {
  graphQLErrors: [
    {
      debugMessage: 'Call to a member function get_head() on bool',
      message: 'Internal server error',
      extensions: [Object],
      locations: [Array],
      path: [Array]
    }
  ]
}`

**Comment by mcourville** [comment]
From what I was able to find so far, `$meta` seems to be empty: https://github.com/ashhitch/wp-graphql-yoast-seo/blob/master/wp-graphql-yoast-seo.php#L758

**Comment by kuuak** [comment]
Having the same issue after upgrade of Yoast to 20.8

**Comment by KhadijaMahanga** [comment]
Having the issue too, since 5 days ago

**Comment by izzygld** [comment]
merged and updated via [#160](https://github.com/ashhitch/wp-graphql-yoast-seo/commit/a98a045035b0c50171c9dbbc43f86d153bcd67ae)


thank you @ashhitch ❤️

**Comment by ashhitch** [comment]
Should now be resolved
